docker-compose.yaml

```diff
@@ -2,6 +2,8 @@ services:
     redis:
         image: redis:alpine
         command: [ redis-server, --maxmemory, 128mb, --maxmemory-policy, volatile-lru, --save, '""' ]
+        volumes:
+            - pimcore-demo-redis-data:/data
```

Comment:
@alexz707 we're using Redis with `--save, '""`, so no data get's persisted, imho having a volume is therefore obsolete, right?

Reply:
Yes you are right, but even with the flag docker generates an anonymous volume and to prevent that I gave it a name.